autoreject/autoreject.py

```diff
@@ -947,7 +947,7 @@ def fit(self, epochs):
         self : instance of AutoReject
             The instance.
         """
-        self.picks_ = _handle_picks(picks=self.picks, info=epochs.info)
+        self.picks_ = _handle_picks(info=epochs.info, picks=self.picks)
```

Comment:
can you revert changes unrelated to the PR?

Reply:
Sure I can, since I changed `_handle_picks`, I went through the files where `_handle_picks` was used; I stumble upon this entry where the arguments are provided in the wrong order, sure with keyword.. but still. It just makes the code a little bit more messy for future improvements in my opinion. Why not improve it together with the fix for this function?

Reply:
I see. The rationale is to keep the diff as clean and small as possible to make the reviewers job easy. For example, I spent a good 30 seconds here wondering if something was broken here and if so, why there was no test for the `Autoreject` class. Until I realized that the two lines were the same ...

Reply:
Ok, I can understand this rationale. However, it also means that to improve existing code, to develop new features, you also need a good 30 seconds to understand why it is written this way. And this is even worst for people which are not familiar with the code of a given repo. In my opinion, this is why there are standards like PEP8 and static analysis tools to improve code readability and help X understand the code written by Y. 
If anything, this confusion was my mistake for not adding to the description of the PR an explanation on this change, even if it was minor.
If you confirm that you want this change revert, I will; but now that you did spend those 30 seconds realizing it was the same code.. wouldn't it be better to keep it as it is in the end an improvement on the former code which might prevent future confusion?

Reply:
Depending on this inclusion or exclusion of bad channels, I will update the different docstring with something similar to MNE.

Reply:
haha let's keep it as is :) General principle is to make cosmetic changes in separate PRs when possible.

Reply:
I'll keep it in mind for future contributions ;)